.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "tests/servers/coordinatord"]
 	path = tests/servers/coordinatord
 	url = https://github.com/revault/coordinatord
+[submodule "tests/servers/cosignerd"]
```

Comment:
update tests/README.md https://github.com/revault/revaultd/tree/master/tests#with-the-servers

Reply:
Good catch, thanks!